benchmark/tmp.yaml

```diff
@@ -0,0 +1,12 @@
+stacks:
+  - name: bucket
```

Comment:
maybe worth having more buckets, and making some buckets rely on other buckets (you can do this explicity with the requires keyword on stacks)